exopy_hqc_legacy/instruments/drivers/visa/yokogawa.py

```diff
@@ -142,7 +142,7 @@ def current(self):
         NB: does not check the current function.
 
         """
-        current = self.query(":SOURce:LEVel?")
+        current = 1e6*self.query(":SOURce:LEVel?")
```

Comment:
The Yokogawa7651 class starts line 305, above that it is all GS200 that was working fine before and not anymore

Reply:
Yes, something weird is going on. All the code was supposed to be for the 7651. I'll check with my instrument tomorrow.